02-cpu-configure/ioc-restart.c

```diff
@@ -0,0 +1,87 @@
+/* BPM IOCs Utils
+ * Copyright (C) 2024 CNPEM
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/*
+ * Slot whitelists to sanitize user input
+ */
+const char* const afc_whitelist[] = {
+    "1", "2", "2-1", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12"
+};
+
+const char* const rffe_whitelist[] = {
+    "11", "12", "13", "14", "15", "16", "17", "18", "19", "20", "21", "22", "23"
+};
```

Comment:
Please, use either `passlist` or `allowlist` instead. Here and elsewhere in the code.

See https://www.kernel.org/doc/html/latest/process/coding-style.html#naming